internal/providers/terraform/google/container_cluster.go

```diff
@@ -0,0 +1,107 @@
+package google
+
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/tidwall/gjson"
+
+	"github.com/shopspring/decimal"
+)
+
+func GetContainerClusterRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "google_container_cluster",
+		RFunc: NewContainerCluster,
+		Notes: []string{
+			"Sustained use discounts are applied to monthly costs, but not to hourly costs.",
+			"Costs associated with non-standard Linux images, such as Windows and RHEL are not supported.",
```

Comment:
Are Windows/RHEL prices available in the Cloud Pricing API? Not suggesting any changes, just wanted to know in case users ask.

Reply:
Yeah the license prices are in the Cloud Pricing API. To add them we'd need to somehow map the images used by the instance to the license that applies.